src/super_gradients/training/losses/seg_kd_loss.py

```diff
@@ -0,0 +1,81 @@
+from typing import Union
+
+import torch
+from torch.nn.modules.loss import _Loss
+
+from super_gradients.training.models.kd_modules.kd_module import KDOutput
+
+
+class SegKDLoss(_Loss):
+    """
+    Wrapper loss for semantic segmentation KD.
+    This loss includes two loss components, `ce_loss` i.e CrossEntropyLoss, and `kd_loss` i.e `CWDKlDivLoss`.
+    """
+
+    def __init__(self, kd_loss: _Loss, ce_loss: _Loss, weights: Union[tuple, list], kd_loss_weights: Union[tuple, list]):
```

Comment:
SegKDLoss uses _Loss as type hints. It is not recommended to use private types (No guarantee _Loss will persist in upcoming releases) and perhaps nn.Module would be a better fit here since you don't use anything specific from _Loss class AFAIK

Reply:
Replaced to nn.Module [CR fixes](https://github.com/Deci-AI/super-gradients/pull/731/commits/59abdf6d228f14d71a74411ad85dac745bdd2efb)